packages/components/src/components/form/Select/index.tsx

```diff
@@ -304,11 +305,6 @@ export const Select = ({
         }
     }, []);
 
-    const closeMenuOnScroll = useCallback(
```

Comment:
Does that mean that the dropdowns stay open when scrolling?

Reply:
Yes... I think we are not losing much if we omit it. It will not immediately close selects near the bottom of a page which was annoying.

Reply:
I think I found a solution. I'd add the close on scroll back and and use `menuPosition="fixed"` instead. Works even better than before :)

Reply:
Nice! A bit annoying that `closeMenuOnScroll` doesn't work with a boolean. It should, according to [docs](https://react-select.com/props). f8becb0c8cb39c673a885d1fd3a13d91f5b3de58